src/main/kotlin/com/hibob/academy/entity/types.kt

```diff
@@ -19,6 +20,19 @@ data class Owner(
     var lastName : String?
 )
 
+data class CreateOwnerRequest(
+    var name: String?,
```

Comment:
Also here is from backward compatibility lesson. We have 2 options for creating a new owner. One with full name. And second option is with first name and last name so because that name field and first name and last name fields are nullable